tests/test_scripts.py

```diff
@@ -2644,7 +2644,8 @@ def replacement_policy(cls, uploader):
         eq_(Mock.mock_policy, policy)
         # There should be two MirrorUploaders for each purpose.
```

Comment:
It would be helpful to clarify this comment while you're at it. I think you meant:
```
    # There should be two MirrorUploaders, one for each purpose.
```

Sorry I missed that on the original commit.

Reply:
Updated the comment